README.md

```diff
@@ -1,14 +1,16 @@
 # DiversiFAQ
 
-Preguntas frecuentes sobre diversidad en tecnología.
+Frecuently asked questions about diversity in tech / Preguntas frecuentes sobre diversidad en tecnología.
 
-Consulta el listado de preguntas [aquí](https://github.com/folkswhocode/diversifaq/blob/master/FAQ.md).
+Check the questions list [here](https://github.com/folkswhocode/diversifaq/blob/master/en/FAQ.md). / Consulta el listado de preguntas [aquí](https://github.com/folkswhocode/diversifaq/blob/master/es/FAQ.md).
 
-**En continua actualización.**
+**In continuous update** / **En continua actualización.**
 
-Para añadir tu pregunta y respuesta, haz una pull request a este repositorio añadiéndola al fichero `FAQ.md`, respetando el formato existente. Y no olvides añadirte al fichero de colaboradores, `CONTRIBUTORS.md`.
```

Comment:
I find this EN/ES structure a little bit confusing tbh 🤔

Reply:
Yes I wasn't sure how to tackle the README.md dual language.. as a third or fourth language would make this option really messy. I thought about having a very simple README.md with just links to each language... and a README.md file under each language folder only with that translation. Do you like that option better or what do you have in mind?